shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1822,6 +1837,53 @@ private AccessibilityEvent obtainAccessibilityEvent(int virtualViewId, int event
     return event;
   }
 
+  /**
+   * Reads the layoutInDisplayCutoutMode value from the window attribute and returns whether a left
+   * cutout insect is required.
+   *
+   * <p>The layoutInDisplayCutoutMode is added after API level 28.
+   */
+  @TargetApi(28)
+  @RequiresApi(28)
+  private boolean doesLayoutInDisplayCutoutModeRequireLeftInsect() {
+    Context context = rootAccessibilityView.getContext();
+    Activity activity = getActivity(context);
+    if (activity != null) {
```

Comment:
if activity is `null` then `doesLayoutInDisplayCutoutModeRequireLeftInsect` is true?

Reply:
I would expect something like this:

```java
if (activity == null || activity.getWindow() == null) {
  // The activity is  null or not not visible, it does not matter whether to apply left insect
  // or not.
  return false;
}
```

Reply:
In that case, it probably doesn't matter? I can set it to false

Reply:
sgtm